exfat_super.c

```diff
@@ -147,8 +147,13 @@ static time_t accum_days_in_year[] = {
 static void _exfat_truncate(struct inode *inode, loff_t old_size);
 
 /* Convert a FAT time/date pair to a UNIX date (seconds since 1 1 70). */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,18,01)
+void exfat_time_fat2unix(struct exfat_sb_info *sbi, struct timespec64 *ts,
+						 DATE_TIME_T *tp)
+#else
 void exfat_time_fat2unix(struct exfat_sb_info *sbi, struct timespec *ts,
 						 DATE_TIME_T *tp)
+#endif
```

Comment:
would be:

```c
#if LINUX_VERSION_CODE < KERNEL_VERSION(4,18,01)
#define timespec64 timespec
#endif
```

a good idea?

Reply:
up to the repo owner I guess

Also this change is because of this commit 
https://github.com/torvalds/linux/commit/95582b00838837fc07e042979320caf917ce3fe6

Which went into kernel v4.18-rc1 so maybe use KERNEL_VERSION(4,18) as the test

You want me to use your change and resubmit the pr?